cafe.rb

```diff
@@ -20,15 +20,16 @@ def take_order(menus)
   end
   print '>'
   order_number = gets.to_i
-  puts "#{menus[order_number][:name]}(#{menus[order_number][:price]}円)ですね。"
-  order_number
+  selected_menu = menus[order_number - 1]
+  puts "#{selected_menu[:name]}(#{selected_menu[:price]}円)ですね。"
+  selected_menu
 end
 
 puts 'bugカフェへようこそ！ご注文は？ 番号でどうぞ'
-order1 = take_order(DRINKS)
+drink_order = take_order(DRINKS) 
```

Comment:
```suggestion
drink_order = take_order(DRINKS)
```

行末の無駄なスペースはなくしましょう。
（エディタの設定で自動的に削除するようにできると思います）